bedrock/mozorg/templates/mozorg/contribute/contribute-tasks.html

```diff
@@ -0,0 +1,176 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "mozorg/contribute/contribute-base.html" %}
+
+{% block body_id %}inquiry{% endblock %}
+
+{% block page_title %}{{_('Get Involved: Volunteer Opportunities at Mozilla')}}{% endblock %}
+
+{% block contrib_nav_cta %}{% endblock %}
+{% block contrib_country_form %}{% endblock %}
+
+{% block contrib_title %}
+  <div class="container">
+    <h1 class="page-title">{{ _('Get involved with Mozilla today!') }}</h1>
+    <p class="section-tagline">{{ _('There are a lots of ways to contribute to Mozilla. Pick one below that you think would be right for you.') }}</p>
+  </div>
+{% endblock %}
+
+{% block contrib_content %}
+  {% if variation == '1' %}
+    <section class="section inquiry">
+      <div class="container inquiry-choices">
+        <div class="option-group">
+          <h2>{{ _('Only have a little time?') }}</h2>
+          <ul class="option-list">
+            <li class="option">
+              <a data-task-id="1" class="ab-task category category-nightly" title="{{ _('Opens new window') }}" href="https://nightly.mozilla.org/" target="_blank">
+                <span class="label-text">{{ _('Help us test by downloading Firefox Nightly') }}</span>
+              </a>
+            </li>
+            <li class="option">
+                <a data-task-id="2" class="ab-task category category-stumbling" title="{{ _('Opens new window') }}" href="https://oneanddone.mozilla.org/tasks/92/" target="_blank">
+                  <span class="label-text">{{ _('Download our Stumbler App') }}</span>
+                </a>
+            </li>
+            <li class="option">
+              <a data-task-id="3" class="ab-task category category-activism" title="{{ _('Opens new window') }}" href="https://sendto.mozilla.org/page/contribute/givenow-seq" target="_blank">
```

Comment:
Could we use donate_url() function here please? This way it will open a localized page when it's available. 
Here's an example https://github.com/mozilla/bedrock/blob/master/bedrock/tabzilla/templates/tabzilla/tabzilla.js#L696